src/software/thunderscope/replay/BUILD

```diff
@@ -34,3 +34,28 @@ py_library(
         ":proto_player",
     ],
 )
+
+py_binary(
+    name = "replay_file_debugging_script",
+    srcs = [
+        "replay_constants.py",
+        "replay_file_debugging_script.py",
+    ],
+    deps = [
+        "//proto:import_all_protos",
+    ],
+)
+
+py_test(
+    name = "replay_corruption_test", 
+    srcs = [
+        'replay_corruption_test.py',
+    ],
+    deps = [
+        "//extlibs/er_force_sim/src/protobuf:erforce_py_proto",
+        "//proto:import_all_protos",
+        "//software/thunderscope:proto_unix_io",
+        ":proto_logger", 
+        ":proto_player"
+    ]
+)
```

Comment:
pytest seems to be a missing dependency here (i.e. `requirement("pytest")`). For that to work you would need to add `load("@simulated_tests_deps//:requirements.bzl", "requirement")` to the top of this BUILD file as well, though since there's already a different `requirement` function imported they're going to clash. Not sure the proper fix for this, but one method could be to add a test folder with its own BUILD file under this directory.